src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,13 +1,50 @@
 # Overview 
 
 Azure.ApplicationModel.Configuration is a component of the .NET Azure SDK. 
-It provides APIs for storing and retrieving application settings.
+It provides APIs for Microsoft [Azure's App Configuration Service](https://docs.microsoft.com/en-us/azure/azure-app-configuration/) which is s a managed configuration service that allows to easily store and manage all application settings in one central place that is separated from code.
+
```

Comment:
what does "managed" means in this context?

Reply:
A service that gives people control (create, update, delete) over the data they need.

Reply:
Ah, this term might be confusing to .NET devs as in .NET "managed" is often used to mean not-native or garbage collected.

Reply:
Ohh yeah you are right, it could lead to that. I will delete it.